packages/@aws-cdk/aws-ecs-patterns/lib/base/network-multiple-target-groups-service-base.ts

```diff
@@ -0,0 +1,433 @@
+import { IVpc } from '@aws-cdk/aws-ec2';
+import { AwsLogDriver, BaseService, CloudMapOptions, Cluster, ContainerDefinition, ContainerImage, ICluster, LogDriver,
+  PropagatedTagSource, Protocol, Secret } from '@aws-cdk/aws-ecs';
+import { NetworkListener, NetworkLoadBalancer, NetworkTargetGroup } from '@aws-cdk/aws-elasticloadbalancingv2';
+import { IRole } from '@aws-cdk/aws-iam';
+import { AddressRecordTarget, ARecord, IHostedZone } from '@aws-cdk/aws-route53';
+import { LoadBalancerTarget } from '@aws-cdk/aws-route53-targets';
+import cdk = require('@aws-cdk/core');
+
+/**
+ * The properties for the base NetworkMultipleTargetGroupsEc2Service or NetworkMultipleTargetGroupsFargateService service.
+ */
+export interface NetworkMultipleTargetGroupsServiceBaseProps {
+  /**
+   * The name of the cluster that hosts the service.
+   *
+   * If a cluster is specified, the vpc construct should be omitted. Alternatively, you can omit both cluster and vpc.
+   * @default - create a new cluster; if both cluster and vpc are omitted, a new VPC will be created for you.
+   */
+  readonly cluster?: ICluster;
+
+  /**
+   * The VPC where the container instances will be launched or the elastic network interfaces (ENIs) will be deployed.
+   *
+   * If a vpc is specified, the cluster construct should be omitted. Alternatively, you can omit both vpc and cluster.
+   * @default - uses the VPC defined in the cluster or creates a new VPC.
+   */
+  readonly vpc?: IVpc;
+
+  /**
+   * The properties required to create a new task definition. One of taskImageOptions or taskDefinition must be specified.
+   *
+   * @default - none
+   */
+  readonly taskImageOptions?: NetworkLoadBalancedTaskImageProps;
+
+  /**
+   * The desired number of instantiations of the task definition to keep running on the service.
+   * The minimum value is 1
+   *
+   * @default 1
+   */
+  readonly desiredCount?: number;
+
+  /**
+   * Name of the service.
+   *
+   * @default - CloudFormation-generated name.
+   */
+  readonly serviceName?: string;
+
+  /**
+   * The period of time, in seconds, that the Amazon ECS service scheduler ignores unhealthy
+   * Elastic Load Balancing target health checks after a task has first started.
+   *
+   * @default - defaults to 60 seconds if at least one load balancer is in-use and it is not already set
+   */
+  readonly healthCheckGracePeriod?: cdk.Duration;
+
+  /**
+   * The network load balancer that will serve traffic to the service. At least one load balancer should be specified.
+   *
+   * @default - a new load balancer with a listener will be created.
```

Comment:
Same as above -- don't need second sentence and maybe clarify only a new one will be created if none provided.